t/plugin/grpc-transcode-struct.t

```diff
@@ -0,0 +1,121 @@
+#
```

Comment:
Could we move these tests under https://github.com/apache/apisix/blob/master/t/plugin/t/plugin/grpc-transcode3.t? This file is small and doesn't have special setup.